src/cascadia/TerminalSettingsEditor/ProfileViewModel.h

```diff
@@ -110,6 +104,12 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         OBSERVABLE_PROJECTED_SETTING(_profile, BellStyle);
         OBSERVABLE_PROJECTED_SETTING(_profile, UseAtlasEngine);
 
+        WINRT_PROPERTY(bool, IsBaseLayer, false);
+        WINRT_PROPERTY(IHostedInWindow, WindowRoot, nullptr);
+        GETSET_BINDABLE_ENUM_SETTING(AntiAliasingMode, Microsoft::Terminal::Control::TextAntialiasingMode, AntialiasingMode);
+        GETSET_BINDABLE_ENUM_SETTING(CloseOnExitMode, Microsoft::Terminal::Settings::Model::CloseOnExitMode, CloseOnExit);
+        GETSET_BINDABLE_ENUM_SETTING(ScrollState, Microsoft::Terminal::Control::ScrollbarState, ScrollState);

```

Comment:
Had to move this down so that it could find the getters/setters defined in the `OBSERVABLE_PROJECTED_SETTING` macros above.

Reply:
Do you want to leave a comment that says "don't shuffle this section up or you're gonna have a bad time?"

Reply:
Eh, if somebody moves them, the errors are pretty clear. So I think it's fine.